checks/collector.py

```diff
@@ -383,19 +369,18 @@ def run(self, checksd=None, start_event=True, configs_reloaded=False):
             if gohai_processes:
                 try:
                     gohai_processes_json = json.loads(gohai_processes)
-                    processes_payload = {
-                        'snaps': [gohai_processes_json.get('processes')],
-                        'format_version': 1
-                    }
-                    if self._is_first_run():
-                        processes_payload['format_description'] = PROCESSES_FORMAT_DESCRIPTION
-
-                    payload['resources'] = {
-                        'processes': processes_payload,
-                        'meta': {
-                            'host': payload['internalHostname'],
+                    processes_snaps = gohai_processes_json.get('processes')
+                    if processes_snaps:
+                        processes_payload = {
+                            'snaps': [processes_snaps]
+                        }
+
+                        payload['resources'] = {
+                            'processes': processes_payload,
+                            'meta': {
+                                'host': payload['internalHostname'],
```

Comment:
Why not use `self.hostname` ? I think it carries more sense than `paylaod['internalHostname]` (while being the same thing).